kornia/testing/__init__.py

```diff
@@ -396,6 +398,25 @@ def KORNIA_CHECK_IS_TENSOR(x, msg: Optional[str] = None):
         raise TypeError(f"Not a Tensor type. Got: {type(x)}.\n{msg}")
 
 
+def KORNIA_CHECK_IS_LIST_OF_TENSOR(x: Optional[Sequence[object]]) -> TypeGuard[List[Tensor]]:
+    """Check the input variable is a List of Tensors.
+
+    Args:
+        x: Any sequence of objects
+
+    Return:
+        True if the input is a list of Tensors, otherwise return False.
+
+    Example:
+        >>> x = torch.rand(2, 3, 3)
+        >>> KORNIA_CHECK_IS_LIST_OF_TENSOR(x)
+        False
+        >>> KORNIA_CHECK_IS_LIST_OF_TENSOR([x])
+        True
+    """
+    return isinstance(x, list) and all(isinstance(d, Tensor) for d in x)
```

Comment:
maybe can be 
```suggestion
    is_list_of_tensor = isinstance(x, list) and all(isinstance(d, Tensor) for d in x)
    if is_list_of_tensor:
        return is_list_of_tensor
    else:
        raise ...
```
to be the same behaviour than the others KORNIA_CHECK -- but to be used with typechecker. we needs the typeguard, which i think needs to return a bool